accounts/admin.py

```diff
@@ -34,5 +14,3 @@ class UserAdmin(BaseUserAdmin):
     }),
 )
 
-admin.site.unregister(User)
```

Comment:
fix my comments on the commit it was mentioned...

Reply:
Does it look ok now? Had some trouble with the rebase